config/software/chef.rb

```diff
@@ -20,9 +20,13 @@
 
 relative_path "chef"
 
+fips_enabled = (project.overrides[:fips] && project.overrides[:fips][:enabled]) || false
```

Comment:
The fact that this works brings equal parts joy and terror to my heart. I wish we could make this more of a top level Omnibus feature rather than abusing an internal data structure (which really needs a more locked down schema).